component-framework/MonacoEditorControl/MonacoEditor/components/editor.tsx

```diff
@@ -0,0 +1,340 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+/* eslint-disable */
```

Comment:
To expand on the question a bit, since these are examples for customers to use, we should be following the standard linting rules that eslint and @typescript-eslint provide.  Any disables should be only for specific reasons and should only be with line comments and not for whole files.